controllers/release/release_adapter.go

```diff
@@ -279,7 +282,7 @@ func (a *Adapter) getTargetReleaseLink() (*v1alpha1.ReleaseLink, error) {
 // registerReleasePipelineRunStatus updates the status of the Release being processed by monitoring the status of the
 // associated release PipelineRun and setting the appropriate state in the Release. If the PipelineRun hasn't
 // started/succeeded, no action will be taken.
-func (a *Adapter) registerReleasePipelineRunStatus(pipelineRun *v1beta1.PipelineRun) error {
+func (a *Adapter) registerReleasePipelineRunStatus(pipelineRun *tekton.ReleasePipelineRun) error {
```

Comment:
See my comment pin `pipeline_run.go` about the new type but if we are using a new type, can't we just change the one last reference to `v1beta1` and remove the import entirely from this file?

Reply:
I don’t really get this comment, sorry. Could you rephrase it?

Reply:
Oh, got you now. I was confused because the discussion was for a different line.

No, I don't think it's possible to do that as a ReleasePipelineRun is not a PipelineRun (it's something more), so listing would not work.

Reply:
I thought the same but figured it was worth asking the go expert

Reply:
I’d like to meet that guy too.